app/models/concerns/event_concern.rb

```diff
@@ -8,16 +8,16 @@ module EventConcern
   IGNORE_RAISE_EVENT_ATTRIBUTES_LIST = %i[availability_status availability_status_error].freeze
 
   IGNORE_RAISE_EVENT_LIST = {
-    "Application"    => IGNORE_RAISE_EVENT_ATTRIBUTES_LIST,
+    "Application"    => IGNORE_RAISE_EVENT_ATTRIBUTES_LIST + %i[_superkey],
     "Authentication" => IGNORE_RAISE_EVENT_ATTRIBUTES_LIST,
     "Endpoint"       => IGNORE_RAISE_EVENT_ATTRIBUTES_LIST
   }.freeze
 
   def ignore_raise_event_for?(attributes)
     ignore_attribute_list = IGNORE_RAISE_EVENT_LIST[self.class.name]
-    return false unless ignore_attribute_list
+    return true unless ignore_attribute_list
```

Comment:
@lindgrenj6 Should not we neg also name of this method in this case?
something like `ignore_raise_event_for` -> `allow_raise_event_or`.
Original intention was - what is in `IGNORE_RAISE_EVENT_LIST` it should be ignored
so If model is not present  in `IGNORE_RAISE_EVENT_LIST` we should not ignore raising event and it means we should raise event.

Reply:
Absolutely, I renamed the method to account for the reversed behavior. Sorry I missed that.